index.html

```diff
@@ -48,41 +48,48 @@ <h1>Welcome to Reelgood</h1>
     </form>
 
     <div class="card" style="display:none">
-        <img src="" id="card1" alt="Card 1">
+       
+        <img src="" id="card1" class="card-img"  alt="Card 1">
         <h3 class="movie" id="title1"></h3>
+        <input class="rating">
         <button class="store-loved-btn">Seen it (Love)</button>
         <button class="store-hated-btn">Seen it (Hate)</button>
         <button class="store-want-btn">Want to see</button>
     </div>
 
     <div class="card" style="display:none">
-        <img src="" id="card2" alt="Card 2">
+        
+        <img src="" id="card2" class="card-img" alt="Card 2">
         <h3 class="movie" id="title2"></h3>
+        <input class="rating">
         <button class="store-loved-btn">Seen it (Love)</button>
         <button class="store-hated-btn">Seen it (Hate)</button>
         <button class="store-want-btn">Want to see</button>
     </div>
 
     <div class="card" style="display:none">
-        <img src="" id="card3" alt="Card 3">
+        <img src="" id="card3" class="card-img" alt="Card 3">
         <h3 class="movie" id="title3"></h3>
+        <input class="rating">
         <button class="store-loved-btn">Seen it (Love)</button>
         <button class="store-hated-btn">Seen it (Hate)</button>
         <button class="store-want-btn">Want to see</button>
     </div>
 
     <div class="card" style="display:none">
-        <img src="" id="card4" alt="Card 4">
-        <h3 class="movie" id="title4"></h3>
+        <img src="" id="card4"class="card-img"  alt="Card 4">
+        <h3 class="movie-image" id="title4"></h3>
+        <input class="rating">
         <button class="store-loved-btn">Seen it (Love)</button>
         <button class="store-hated-btn">Seen it (Hate)</button>
         <button class="store-want-btn">Want to see</button>
     </div>
 
     <!-- Card 5 -->
     <div class="card" style="display:none">
-        <img src="" id="card5" alt="Card 5">
+        <img src="" id="card5" class="card-img"  alt="Card 5">
         <h3 class="movie" id="title5"></h3>
+        
         <button class="store-loved-btn">Seen it (Love)</button>
```

Comment:
why does this last card not have a rating input?

Reply:
i think i just forgot to add them since i was trying to think of what i could do